hayeon/programmers-2/프린터.js

```diff
@@ -0,0 +1,23 @@
+function solution(priorities, location) {
+  let ranking = 0;
+
+  while (priorities.length) {
+    if (priorities[0] < Math.max(...priorities)) {
+      if (--location < 0) location = priorities.length - 1;
+      priorities.push(priorities.shift());
+    } else {
+      ranking++;
+      if (--location < 0) return ranking;
+      priorities.shift();
+    }
+  }
+}
```

Comment:
```suggestion
function solution(priorities, location) {
  let ranking = 0;

  while (priorities.length) {
    const isMaxPriority = priorities[0] < Math.max(...priorities);

    if (isMaxPriority) {
      if (--location < 0) {
        location = priorities.length - 1;
      }
      priorities.push(priorities.shift());
      continue;
    }
    if (--location < 0) return ranking + 1;

    ranking++;
    priorities.shift();
  }
}
```
- 풀이잘봤습니다!
- 블록문을 최소화해서 좀 더 가독성좋게 수정해봤습니다! 참고만해주세요